src/main/controllers/price-service.js

```diff
@@ -61,12 +64,39 @@ const loadPriceData = async () => {
 	emitter.emit('pricesUpdated', dataToInsert);
 };
 
+const startStream = async () => {
+	const socket = io('https://coincap.io');
+
+	socket.on('trades', async tradeMsg => {
+		const tokenData = tradeMsg.message.msg;
+		if (existing[tokenData.short] && existing[tokenData.short].price !== tokenData.price) {
+			const btcPriceUsd = await TokenPrice.findBySymbol('BTC');
+			const ethPriceUsd = await TokenPrice.findBySymbol('ETC');
+
+			const tokenData = tradeMsg.message.msg;
+
+			const tokenPrice = {
+				id: existing[tokenData.short].id,
+				name: tokenData.long,
+				symbol: tokenData.short,
+				source: 'https://coincap.io',
+				priceUSD: +tokenData.price,
+				priceBTC: +tokenData.price / btcPriceUsd,
+				priceETH: +tokenData.price / ethPriceUsd,
+				updatedAt: Date.now()
+			};
+			await TokenPrice.bulkEdit([tokenPrice]);
+
+			emitter.emit('pricesUpdated', await TokenPrice.findAll());
```

Comment:
probably need to change pricesUpdated logic to allow sending only one symbole, no sense in sending all of them in each change